frequency-chain/frequency-xyz
# Contact form

# Description

- Closes #56

## Screenshot(s)


## Type of change
      re-approval.)

# How to Test?
Because the "goto" takes you to "/" which is frequency.xyz/ and not the PR link, to verify the form clearing behavior you can try it locally.

You also may need to allow site trackers.

1. 
```shell
git fetch && git checkout contact-section-#56
npm run dev # this disables actually POSTing to the endpoint and will always succeed.
```
2. Click on 'Contact' in the menu bar
3. Click submit and see that the error fields are marked.
4. Fill it out partially, click submit and see that the unfilled required fields are marked
5. Fill it out entirely & click submit, see that you get the "Thank you" card.
6. Click the card and you should be 
   a.  scrolled to the top
   b.  if you scroll back down, the form should be cleared.

# Checklist:
